sphinx/types.rst

```diff
@@ -1,104 +1,168 @@
 .. types:
 
-Project Data
-==================
-
-New data imported into the openoa toolkit can take advantage of the data structures in the `types` module.
-The base class is openoa.types.PlantData, which represents all known pieces of data about a given wind plant.
-
-
-Schemas
+Plant Data
+##########
+
+New data imported into the openoa toolkit can take advantage of the data structures in the :py:mod:`plant`
+module. As of v3, users can use the :py:class:`openoa.plant.PlantData` class, instead of subclassing :py:class:`PlantData`,
+and implementing their own :py:meth:`prepare()` method. Additionally, as of v3, :py:class:`openoa.plant.PlantData` is an attrs
+dataclass, and each of the core representations are Pandas :py:class:`DataFrame`s, e.g., :py:attr:`openoa.plant.PlantData.scada`,
+:py:attr:`openoa.plant.PlantData.reanalysis`, etc.
+
+The added benefit of adopting attrs for the core structure is many data checks can happen automatically
```

Comment:
I think talking about attrs is inappropriate here, since to the average user this is more of an implementation detail.

Reply:
Agree that discussing attrs probably belongs later in the section after new users get a higher-level overview of how to use the PlantData class

Reply:
I've switched it up to have more general details for users in this section.